lib/pre-build.js

```diff
@@ -371,6 +371,17 @@ function parseArgs(_argv, _opts) {
 }
 
 function executeCli(cmd, target) {
+  /* eslint-disable no-console */
+  console.log(
+    [
+      '============================================================================\n',
+      `Attempting ${cmd} in native-metrics module. Please note that this is an`,
+      '\n OPTIONAL dependency, and any resultant errors in this process will not',
+      '\n affect the performance of the New Relic agent itself.',
```

Comment:
This statement is a bit untrue. Without this module the agent will not get event loop or garbage collection metrics. Not sure if this is really important, but it does feel like it could be misleading.

Separately, why is the newline at the end of the first line but at the beginning of the other lines? Consistency is good, and I'd vote for changing the `.join()` string to `'\n'` rather than including the newlines manually.